tachyon/math/base/rational_field.h

```diff
@@ -101,7 +101,11 @@ class RationalField : public Field<RationalField<F>> {
     return numerator_ * other.denominator_ >= other.numerator_ * denominator_;
   }
 
-  F Evaluate() const { return numerator_ / denominator_; }
+  F Evaluate() const {
+    const std::optional<F> div = numerator_ / denominator_;
+    CHECK(div);
+    return *div;
```

Comment:
I think we should change it to like below, but i'm not 100% sure, so please could you also check that this will cause move operation?

```suggestion
    return std::move(div).value();
```

Reply:
This indeed invokes the move operation properly!